example_pkg/.spin/cmds.py

```diff
@@ -14,7 +16,7 @@ def example(flag):
     click.secho("Running example custom command", bold=True, fg="bright_blue")
     print()
     config = util.get_config()
-    commands = util.get_commands()
```

Comment:
This is for illustrative purposes, so would like to leave it in. Maybe I should just print the commands out.